test/integration/server-side-dev-errors/test/index.test.js

```diff
@@ -328,6 +328,8 @@ describe('server-side dev errors', () => {
              |   ^",
            "stack": [
              "handler pages/api/hello.js (2:3)",
+             "async invokeRouteModule ../../../packages/next/dist/src/build/templates/pages-api.ts (184:7)",
```

Comment:
We should be able to skip the ignored frames (next internals) with `toDisplayRedbox` cc @eps1lon

Reply:
These aren't ignore-listed in this specific tests since it's an integration test that runs with a symlinked Next.js which will not point into `node_modules`. We have this in another place as well. 

I'll see if I can come up with some logic to hide symlinked Next.js for integration tests.

Might also be fixed by ignore-listing transpiled files instead.

Reply:
Might go away once rebased onto https://github.com/vercel/next.js/pull/78242. But no guarantee.

Reply:
Looks like that did it, it passes without the internal frames now